src/_ert/forward_model_runner/cli.py

```diff
@@ -71,26 +79,26 @@ def _setup_logging(directory: str = "logs"):
 JOBS_JSON_RETRY_TIME = 30
 
 
-def _wait_for_retry():
-    time.sleep(JOBS_JSON_RETRY_TIME)
+async def _wait_for_retry():
```

Comment:
Wondering if we need this helper function at al?

Reply:
Yes, we need it for one of the tests. `test_job_dispatch.py::test_retry_of_jobs_json_file_read`

Reply:
Hm, this usage of that function is a bit strange though.

Reply:
We mock it to `lock.acquire` in a test, so that it will stop here